### ISSUE TITLE
Print out email

### ISSUE THREAD

**Issue by lp-SRO** [nextcloud/mail#8866]
### Is your feature request related to a problem? Please describe.

Is it not possible to print out the contents of an email? Even in today's times, this is still necessary from time to time.

### Describe the solution you'd like

You should add a printer symbol at the top of an email. For example, in front of the Symbohl wastebasket. You can then use this symbol to print an email.

### Additional context

Thank you for your work :-)

**Comment by ChristophWurst** [comment]
Have you tried printing the page? There are special print stylesheets in place to make the contents of an email printable.

**Comment by lp-SRO** [comment]
Where can I find these?

**Comment by ChristophWurst** [comment]
Ctrl+P

**Comment by lp-SRO** [comment]
Yes, but then only what you can see will be printed. If the content is longer and it becomes two pages, it won't print it.
That doesn't make any sense at all. Basically it just prints out a screenshot.

**Comment by BiglifeMatt** [comment]
Print preview would certainly be desired by my end users.

**Comment by handkerchief333** [comment]
There is no need to discuss whether this is a useful feature. In fact, it is an extremely useful and necessary feature. So much so that you almost have to use a different e-mail program because it's missing. So +1
See #9301 for my 2 cents.
